pkg/skaffold/deploy/error/errors.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy"
```

Comment:
it seems a bit odd that the `error` package is taking a dependency on the `deploy` package and not the other way round.

Reply:
Just making sure you did not confuse `skaffold/error` package with  `deploy/error`.


That said, I should be able to revert the dependency. (hopefully no cyclic dependency)

Reply:
I had seen that; it seemed at the moment that `deploy` should import `deploy/error` but now I don't have a strong opinion either way.

Reply:
fixed!